Void linux installation instructions are mising libXinerama-devel

https://ebitengine.org/en/documents/install.html

the instructions for void linux are missing `libXinerama-devel`

without it we get the following error
```bash
go run .
# github.com/hajimehoshi/ebiten/v2/internal/glfw
In file included from internal_unix.h:96,
                 from context_unix.c:8:
x11_platform_linbsd.h:25:10: fatal error: X11/extensions/Xinerama.h: No such file or directory
   25 | #include <X11/extensions/Xinerama.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/contents/en/documents/install.html
+++ b/contents/en/documents/install.html
@@ -52,7 +52,7 @@
     <h3>Alpine</h3>
     <pre><code>sudo apk add alsa-lib-dev libx11-dev libxrandr-dev libxcursor-dev libxinerama-dev libxi-dev mesa-dev pkgconf</code></pre>
     <h3>Void</h3>
-    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config</code></pre>
+    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config libXinerama-devel</code></pre>
   </div>
   <div data-tags="freebsd">
     <pre><code>pkg install alsa-lib libxcursor libxi libxinerama libxrandr mesa-libs pkgconf</code></pre>

--- a/contents/en/documents/install.html
+++ b/contents/en/documents/install.html
@@ -52,7 +52,7 @@
     <h3>Alpine</h3>
     <pre><code>sudo apk add alsa-lib-dev libx11-dev libxrandr-dev libxcursor-dev libxinerama-dev libxi-dev mesa-dev pkgconf</code></pre>
     <h3>Void</h3>
-    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config libXinerama-devel</code></pre>
+    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXinerama-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config</code></pre>
   </div>
   <div data-tags="freebsd">
     <pre><code>pkg install alsa-lib libxcursor libxi libxinerama libxrandr mesa-libs pkgconf</code></pre>

--- a/contents/ja/documents/install.html
+++ b/contents/ja/documents/install.html
@@ -52,7 +52,7 @@
     <h3>Alpine</h3>
     <pre><code>sudo apk add alsa-lib-dev libx11-dev libxrandr-dev libxcursor-dev libxinerama-dev libxi-dev mesa-dev pkgconf</code></pre>
     <h3>Void</h3>
-    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config</code></pre>
+    <pre><code>sudo xbps-install libXcursor-devel libXi-devel libXinerama-devel libXrandr-devel libXxf86vm-devel MesaLib-devel pkg-config</code></pre>
   </div>
   <div data-tags="freebsd">
     <pre><code>pkg install alsa-lib libxcursor libxi libxinerama libxrandr mesa-libs pkgconf</code></pre>

